<ui version="4.0" >
 <class>AddChannelDialog</class>
 <widget class="QDialog" name="AddChannelDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>RSS Channel URL</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>RSS Link:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leRssLink" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>146</red>
           <green>145</green>
           <blue>144</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>165</red>
           <green>164</green>
           <blue>164</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="text" >
      <string>http://</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>RSS Alias:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="leRssAlias" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>85</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>85</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>165</red>
           <green>164</green>
           <blue>164</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string>RSS</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>178</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddChannelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel" >
     <x>23</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AddChannelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>351</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel" >
     <x>294</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
